src/charm.py

```diff
@@ -54,6 +57,11 @@ def __init__(self, *args) -> None:
             self.on.slurm_controller_relation_joined, self._slurm_controller_relation_joined
         )
 
+        self.service_init_sync("auth-munge", False)
+        self.service_init_sync("slurm-compute", False)
+        self.service_init_sync("slurm-controller", False)
```

Comment:
Are you wanting to locally track the sync state of these?

Even if you did, the `slurm-compute` one would not need it.

Reply:
Let me know if I am getting this right - would I want to track the sync state of `slurm-controller` and `auth-munge` since those relations are needed for the client and server to work correctly with one another? I would not need `slurm-compute` however since that is just sharing discovery information about the node?

Reply:
For the server, syncs may be kept for the local controller config and munge key. But they would not be tied to a relation.

The `slurm-compute` sync could be dropped because it wouldn't actually indicate anything other than at least slurm-client on a compute node is attached, which doesn't seem helpful. Neither would it be a "required" sync that affects slurm server status.